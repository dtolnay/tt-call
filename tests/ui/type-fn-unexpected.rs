use tt_call::{parse_type, tt_call};

tt_call! {
    macro = [{ parse_type }]
    input = [{ fn f }]
}

fn main() {}
