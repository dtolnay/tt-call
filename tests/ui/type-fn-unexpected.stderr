error: no rules expected the token `f`
 --> tests/ui/type-fn-unexpected.rs:5:19
  |
5 |     input = [{ fn f }]
  |                   ^ no rules expected this token in macro call
  |
  = note: while trying to match end of macro
